File: 5089 - Programação – Algoritmos/heranca_java/src/Pessoa.java
public class Pessoa {

    String nome;
    int idade;

    public Pessoa(String nome, int idade) {
        this.nome = nome;
        this.idade = idade;
    }

    public void funcPessoa() {
        System.out.println("funcPessoa");
    }


    @Override
    public String toString() {
        return "Pessoa";
    }
}

File: 5424 - Acesso móvel a sistemas de informação/doird/Kotlin/2_classes/src/Pessoa.kt
open class Pessoa {

    var nome: String
    var idade: Int

    constructor(nome: String, idade: Int){
        this.nome = nome
        this.idade = idade
    }

    init {
        println("o obj foi Criado")
    }

    private fun getData(): String{
        return "nome:${this.nome}\nidade:${this.idade}"
    }


    override fun equals(other: Any?): Boolean {
        if (other is Pessoa){
           return this.nome.length == other.nome.length + 3
        }
        return false
    }

    override fun toString(): String {
        return this.getData()
    }
}